biocrnpyler/units.py

```diff
@@ -0,0 +1,164 @@
+import libsbml
```

Comment:
please add a license header.

Reply:
> 
> 
> It looks good to me, but I'm a bit confused by these helper functions. There are lots of them and hard to maintain them.
> 
> Something like this would be a bit simpler:
> 
> ```python
> valid_units = {'litre_per_mole_per_hour': {unit_kin: [libsbml.UNIT_KIND_SECOND, libsbml.UNIT_KIND_LITRE, libsbml.UNIT_KIND_MOLE], 
>         unit_exponents:[-1,1,-1], unit_scale:[0,0,0], unit_multipler:[3600,1,1]}}
> ```
> 
> and
> 
> ```python
> def create_new_unit_definition(model, user_input):
>     check_user_input(user_input)
>     # create new unit
> 
> def check_user_input(user_input):
>     user_input in valid_units
> ```

Yep, this is a good idea, will do something like that.